sopel/irc/capabilities.py

```diff
@@ -43,12 +109,22 @@ def get_capability_info(self, name: str) -> CapabilityInfo:
 
     @property
     def available(self) -> Dict[str, Optional[str]]:
-        """Dict of available server capabilities."""
+        """Dict of available server capabilities.
+
+        Each key is the name of a capability advertised by the server, and each
+        key is the parameters as advertised (if any) for this capability.
```

Comment:
```suggestion
        value is the parameters as advertised (if any) for this capability.
```

I assume this is not a special dict in which each item has two keys and no values 😉